src/scripts/main.js

```diff
@@ -1,3 +1,22 @@
 'use strict';
 
-// write code here
+const listItems = [...document.querySelectorAll('li')];
+const list = document.querySelector('ul');
+
+function sortList(li) {
+  function toNumber(salary) {
+    return +salary.slice(1).split(',').join('');
+  }
```

Comment:
Try to avoid nested functions
```suggestion
function toNumber(salary) {
  return +salary.slice(1).split(',').join('');
}
 
function sortList(li) {
```